ubuntu1704-desktop.json

```diff
@@ -0,0 +1,15 @@
+{
+  "_comment": "Build with `packer build -var-file=ubuntu1704-desktop.json ubuntu.json`",
+  "vm_name": "ubuntu1704-desktop",
+  "desktop": "true",
+  "locale": "en_US.UTF-8",
+  "cpus": "1",
+  "disk_size": "130048",
```

Comment:
Not shady. I'm new to vagrant. And, just now learning about this project. So, is there a forum or other place to discuss and learn if that value is the size of vm or size of iso?